code/game/objects/items/ego_weapons/he.dm

```diff
@@ -73,7 +78,7 @@
 			if(L == user || ishuman(L))
 				continue
 			L.apply_damage(aoe, BLACK_DAMAGE, null, L.run_armor_check(null, BLACK_DAMAGE), spread_damage = TRUE)
-
```

Comment:
While we're at it:
`for(var/mob/living/L in livinginrange(1, user))`
livinginrange is to be used for very long ranges, as it will iterate through the list of all living mobs, thus rendering it much more resource intensive for very low distances(below ~24).

Reply:
Understandable. I was never told how it worked exactly.